scripts/deploy

```diff
@@ -129,13 +126,8 @@ $python -m pip install --upgrade pip
 $python -m pip install poetry
 echo -e "Finished checking dependencies\n"
 
-if ! $mock_ctrl; then
-  # set ENABLE_HW flag since this is being deployed to a machine with the actual hardware setup
-  sed -i 's/DISABLE_HW = True/DISABLE_HW = False/' ../amplipi/rt.py
-fi
-
 # use a new version for the build but hide it from git since it isn't a real release
-old_version=$(poetry version -s)
+base_version=$(poetry version -s)
```

Comment:
```suggestion
base_version=$(poetry version -s | sed 's/+.*//') # trim +XXXXX to avoid generating a bad version below
old_version=$(poetry version -s)
```

Reply:
This is not really necessary, but handles the case where this script failed building before setting the version back